### PR TITLE
[IMP] productivity: calendar add troubleshooting

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=3650100&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@jev-odoo 

Can you review the changes on this PR when you have a moment? 

Outlook.rst
Lines: 14-182

Google.rst
Lines: 157-197

Thanks!

**Comment by tiku-odoo** [comment]
@jero-odoo 

Can you peer reivew these two doc changes: 
Outlook.rst
Lines: 14-182

Google.rst
Lines: 157-197

Thanks,
Tim :+1:

File: content/applications/productivity/calendar/outlook.rst
Comment (jero-odoo) [inline content/applications/productivity/calendar/outlook.rst:144]:
```suggestion
Sync issues can be seen in the database logs. In these cases, the account needs
```

File: content/applications/productivity/calendar/google.rst
Comment (jero-odoo) [inline content/applications/productivity/calendar/google.rst:196]:
```suggestion
Click :guilabel:`Confirm` after making the selection about modifying the user's events and the
```

File: content/applications/productivity/calendar/outlook.rst
Comment (jero-odoo) [inline content/applications/productivity/calendar/outlook.rst:180]:
```suggestion
Click :guilabel:`Confirm` after making the selection about modifying the user's events and the
```

File: content/applications/productivity/calendar/outlook.rst
Comment (jero-odoo) [inline content/applications/productivity/calendar/outlook.rst:178]:
```suggestion
 - :guilabel:`Leave them untouched`: no changes to the events.
 - :guilabel:`Delete from the current Microsoft Calendar account`: delete the events from Microsoft
   calendar.
 - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
 - :guilabel:`Delete from both`: delete the events from both: Microsoft calendar and Odoo calendar.

 :guilabel:`Next Synchronization`:

 - :guilabel:`Synchronize only new events`: sync new events on either/both: Microsoft calendar, Odoo
   calendar.
 - :guilabel:`Synchronize all existing events`: sync all events on either/both Microsoft calendar,
   Odoo calendar.
```

File: content/applications/productivity/calendar/google.rst
Comment (jero-odoo) [inline content/applications/productivity/calendar/google.rst:194]:
```suggestion
 - :guilabel:`Leave them untouched`: no changes to the events.
 - :guilabel:`Delete from the current Google Calendar account`: delete the events from Google
   Calendar.
 - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
 - :guilabel:`Delete from both`: delete the events from both: Google Calendar and Odoo calendar.

 :guilabel:`Next Synchronization`:

 - :guilabel:`Synchronize only new events`: sync new events on either/both: Google Calendar, Odoo
   calendar.
 - :guilabel:`Synchronize all existing events`: sync all events on either/both Google Calendar, Odoo
   calendar.
```

**Comment by jev-odoo** [comment]
@tiku-odoo I assume you meant @jevs-odoo who created the task ?

**Comment by tiku-odoo** [comment]
> @tiku-odoo I assume you meant @jevs-odoo who created the task ?

@jev-odoo Yes, sorry about the mis-tagging.

:+1:

**Comment by tiku-odoo** [comment]
@jevs-odoo 

Can you review the changes on:
Outlook.rst
Lines: 140-182

Google.rst
Lines: 157-197

Thanks!

**Comment by tiku-odoo** [comment]
@ksc-odoo 

Can you review this doc when you have a moment?

Thanks,
Tim :+1:

File: content/applications/productivity/calendar/google.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/google.rst:164]:
```suggestion
There may be times when the *Google Calendar* account does not sync correctly with Odoo. Sync issues
can be seen in the database logs.

In these cases, the account needs troubleshooting. A reset can be performed using the
:guilabel:`Reset Account` button, which can be accessed by navigating to :menuselection:`Settings
app --> Manage Users`. Then, select the user to modify the calendar, and click the
:guilabel:`Calendar` tab.
```

File: content/applications/productivity/calendar/google.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/google.rst:175]:
```suggestion
The following reset options are available for troubleshooting *Google Calendar* sync with Odoo:
```

File: content/applications/productivity/calendar/google.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/google.rst:193]:
```suggestion
 - :guilabel:`Synchronize all existing events`: sync all events on either/both: *Google Calendar*, Odoo
```

File: content/applications/productivity/calendar/outlook.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/outlook.rst:147]:
```suggestion
There may be times when the *Microsoft Outlook Calendar* account does not sync correctly with Odoo.
Sync issues can be seen in the database logs. 

In these cases, the account needs troubleshooting. A reset can be performed using the
:guilabel:`Reset Account` button, which can be accessed by navigating to :menuselection:`Settings
app --> Manage Users`. Then, select the user to modify the calendar, and click on the
:guilabel:`Calendar` tab.
```

File: content/applications/productivity/calendar/outlook.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/outlook.rst:158]:
```suggestion
The following reset options are available for troubleshooting *Microsoft Outlook Calendar* sync with
```

Comment (ksc-odoo) [reply content/applications/productivity/calendar/outlook.rst:158]: suggesting to capitalize *through* calendar because that's how Microsoft refers to it on their official website: https://support.microsoft.com/en-us/office/introduction-to-the-outlook-calendar-d94c5203-77c7-48ec-90a5-2e2bc10bd6f8

File: content/applications/productivity/calendar/outlook.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/outlook.rst:171]:
```suggestion
 - :guilabel:`Delete from the current Microsoft Calendar account`: delete the events from *Microsoft
   Outlook Calendar*.
 - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
 - :guilabel:`Delete from both`: delete the events from both: *Microsoft Outlook Calendar* and Odoo calendar.
```

File: content/applications/productivity/calendar/google.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/google.rst:187]:
```suggestion
 - :guilabel:`Delete from the current Google Calendar account`: delete the events from *Google
   Calendar*.
 - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
 - :guilabel:`Delete from both`: delete the events from both: *Google Calendar* and Odoo calendar.
```

File: content/applications/productivity/calendar/google.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/google.rst:191]:
```suggestion
 - :guilabel:`Synchronize only new events`: sync new events on either/both: *Google Calendar*, Odoo
```

File: content/applications/productivity/calendar/outlook.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/outlook.rst:178]:
```suggestion
 - :guilabel:`Synchronize only new events`: sync new events on either/both: *Microsoft Outlook Calendar*, Odoo
   calendar.
 - :guilabel:`Synchronize all existing events`: sync all events on either/both: *Microsoft Outlook Calendar*,
   Odoo calendar.
```

File: content/applications/productivity/calendar/outlook.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/outlook.rst:180]:
```suggestion
Click :guilabel:`Confirm` after making the selection to modify the user's events and the
```

File: content/applications/productivity/calendar/google.rst
Comment (ksc-odoo) [inline content/applications/productivity/calendar/google.rst:196]:
```suggestion
Click :guilabel:`Confirm` after making the selection to modify the user's events and the
```

**Review by ksc-odoo — APPROVED** [review]
Hey @tiku-odoo -- just finished my Final Review. Awesome job! Approving now. There is only a handful of minor adjustments that require your attention. Once those modifications have been completed, feel free to tag this for Tech Review. Thanks! 👍

**Comment by tiku-odoo** [comment]
@StraubCreative 

This doc is ready for your technical review when you have a moment. 

Thanks, Tim

File: content/applications/productivity/calendar/google.rst
Comment (StraubCreative) [inline content/applications/productivity/calendar/google.rst:189]: same here, the nested colon is strange choice of punctuation to me

File: content/applications/productivity/calendar/outlook.rst
Comment (StraubCreative) [inline content/applications/productivity/calendar/outlook.rst:181]: nested colons here, as well

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @tiku-odoo 
A couple quick comments below.
As well, there are lines that are either too long or too short in both docs. Please address them both before merge, thanks!

**Comment by tiku-odoo** [comment]
@StraubCreative 

Thanks for the review. I've made all the changes you requested. 

Ready for merge. 

Thanks, 
Tim :+1:

**Review by StraubCreative — APPROVED** [review]
Looks good thanks @tiku-odoo 
..
@robodoo r+

**Comment by tiku-odoo** [comment]
@robodoo retry

**Comment by StraubCreative** [comment]
@robodoo retry

**Pull request closed without merging**
